data/helpers.d/package

```diff
@@ -35,15 +35,15 @@ ynh_apt() {
 #
 # usage: ynh_package_update
 ynh_package_update() {
-    ynh_apt update
+    ynh_apt --no-remove update
```

Comment:
but during `install` one can remove a package. If you read apt's man page, you can see that 
> If a hyphen is appended to the package name (with no intervening space), the identified package will be removed if it is installed.
( http://manpages.ubuntu.com/manpages/xenial/en/man8/apt-get.8.html )

Stupid ! but we should take care of everything !